pkg/repository/repository.go

```diff
@@ -134,6 +138,30 @@ func (r FFRepository) SetFlag(featureConfig rest.FeatureConfig, initialLoad bool
 	}
 }
 
+// SetFlags places all the flags in the repository
+func (r FFRepository) SetFlags(initialLoad bool, envID string, featureConfigs ...rest.FeatureConfig) {
```

Comment:
It would probably be worth trying to add a new test for this function.   I guess we can test that when SetFlags is called, that the cache is populated and the callback is called?

There might be a bit of a pain setting this up if a test doesn't already exist, but it feels like the kind of thing that might break in the future and we'd easily miss it.